src/modal-manager.ts

```diff
@@ -75,14 +80,19 @@ export class ModalManager extends LitElement implements ModalManagerInterface {
     }
   }
 
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.modal.deactivate();
+  }
+
   /** @inheritdoc */
   render(): TemplateResult {
     return html`
       <div class="container">
         <div class="backdrop" @click=${this.backdropClicked}></div>
         <modal-template
           @closeButtonPressed=${this.closeButtonPressed}
-          tabindex="0"
+          tabindex="-1"
```

Comment:
👍

Q: is this equivalent to just not including a `tabindex` on this element at all?

Reply:
It allows JS focus() but removes the element from tab order.
I had removed the `tabindex` attribute completely, but tests showed it should be in the container.